lib/ejs.js

```diff
@@ -69,6 +69,17 @@ var _BOM = /^\uFEFF/;
 exports.cache = utils.cache;
 
 /**
+ * For security reasons, Sometimes the template file may have some
+ * pre-processing.For example, inserts some checking data in the form of
+ * time reading the html file.
+ *
+ * @type {Function}
+ * @public
```

Comment:
Should reference a custom fileLoader type, with a type definition in [`docs/jsdoc`](https://github.com/mde/ejs/tree/master/docs/jsdoc).